src/main/java/uk/gov/hmcts/reform/iacasenotificationsapi/domain/entities/ContactPreference.java

```diff
@@ -0,0 +1,44 @@
+package uk.gov.hmcts.reform.iacasenotificationsapi.domain.entities;
+
+
+import static java.util.Arrays.stream;
+
+import com.fasterxml.jackson.annotation.JsonValue;
+import java.util.Optional;
+
+public enum ContactPreference {
```

Comment:
Do we actually need this `ContactPreference` class? Does it help us in any way compared to just reading which value is present. Perhaps further discussion so our tickets align.

Reply:
I think we need it in order to decide if the appellant prefers SMS or EMAIL, this will prevent sending both notification in case the email and mobile number are populated.